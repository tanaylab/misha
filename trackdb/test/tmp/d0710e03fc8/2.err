bash: module: line 1: syntax error: unexpected end of file
bash: error importing function definition for `BASH_FUNC_module'
bash: ml: line 1: syntax error: unexpected end of file
bash: error importing function definition for `BASH_FUNC_ml'
Loading required package: sc5mc
Loading required package: gpatterns

Attaching package: ‘gpatterns’

The following objects are masked _by_ ‘.GlobalEnv’:

    fread, qqv

Loading required package: bindrcpp
Joining, by = "index1"
Joining, by = "index2"
Joining, by = c("index1", "index2", "illumina_index", "lib")
0.00reads [00:00, ?reads/s]51.9Kreads [00:00, 519Kreads/s]114Kreads [00:00, 545Kreads/s] 177Kreads [00:00, 569Kreads/s]240Kreads [00:00, 585Kreads/s]308Kreads [00:00, 612Kreads/s]374Kreads [00:00, 626Kreads/s]434Kreads [00:00, 618Kreads/s]505Kreads [00:00, 642Kreads/s]573Kreads [00:00, 654Kreads/s]642Kreads [00:01, 662Kreads/s]708Kreads [00:01, 662Kreads/s]778Kreads [00:01, 674Kreads/s]851Kreads [00:01, 690Kreads/s]921Kreads [00:01, 692Kreads/s]990Kreads [00:01, 676Kreads/s]
0.00reads [00:00, ?reads/s]70.5Kreads [00:00, 705Kreads/s]141Kreads [00:00, 703Kreads/s] 212Kreads [00:00, 706Kreads/s]286Kreads [00:00, 714Kreads/s]355Kreads [00:00, 709Kreads/s]427Kreads [00:00, 710Kreads/s]501Kreads [00:00, 719Kreads/s]574Kreads [00:00, 722Kreads/s]643Kreads [00:00, 714Kreads/s]712Kreads [00:01, 694Kreads/s]785Kreads [00:01, 705Kreads/s]859Kreads [00:01, 714Kreads/s]930Kreads [00:01, 715Kreads/s]
0.00reads [00:00, ?reads/s]48.0Kreads [00:00, 479Kreads/s]86.3Kreads [00:00, 446Kreads/s]120Kreads [00:00, 407Kreads/s] 165Kreads [00:00, 408Kreads/s]195Kreads [00:00, 350Kreads/s]243Kreads [00:00, 381Kreads/s]282Kreads [00:00, 382Kreads/s]318Kreads [00:00, 368Kreads/s]370Kreads [00:00, 404Kreads/s]410Kreads [00:01, 379Kreads/s]461Kreads [00:01, 409Kreads/s]502Kreads [00:01, 410Kreads/s]544Kreads [00:01, 377Kreads/s]598Kreads [00:01, 415Kreads/s]648Kreads [00:01, 438Kreads/s]698Kreads [00:01, 454Kreads/s]745Kreads [00:01, 455Kreads/s]808Kreads [00:01, 496Kreads/s]859Kreads [00:02, 350Kreads/s]901Kreads [00:07, 26.2Kreads/s]955Kreads [00:07, 36.7Kreads/s]992Kreads [00:25, 6.09Kreads/s]
0.00reads [00:00, ?reads/s]831reads [00:00, 3.01Kreads/s]3.77Kreads [00:00, 4.06Kreads/s]58.0Kreads [00:00, 5.74Kreads/s]126Kreads [00:00, 8.18Kreads/s] 193Kreads [00:00, 11.6Kreads/s]264Kreads [00:01, 16.5Kreads/s]332Kreads [00:01, 23.3Kreads/s]403Kreads [00:01, 32.8Kreads/s]475Kreads [00:01, 46.0Kreads/s]545Kreads [00:01, 63.9Kreads/s]617Kreads [00:01, 88.0Kreads/s]687Kreads [00:01, 119Kreads/s] 760Kreads [00:01, 159Kreads/s]832Kreads [00:01, 208Kreads/s]902Kreads [00:01, 262Kreads/s]971Kreads [00:02, 319Kreads/s]
0.00reads [00:00, ?reads/s]11.9Kreads [00:00, 118Kreads/s]15.4Kreads [00:00, 20.8Kreads/s]40.8Kreads [00:00, 28.7Kreads/s]50.3Kreads [00:01, 29.0Kreads/s]57.8Kreads [00:01, 28.1Kreads/s]89.4Kreads [00:01, 38.7Kreads/s]103Kreads [00:01, 46.3Kreads/s] 115Kreads [00:01, 56.1Kreads/s]127Kreads [00:01, 66.6Kreads/s]140Kreads [00:01, 77.4Kreads/s]152Kreads [00:02, 75.2Kreads/s]162Kreads [00:02, 67.0Kreads/s]171Kreads [00:02, 57.7Kreads/s]179Kreads [00:02, 62.9Kreads/s]187Kreads [00:02, 37.3Kreads/s]193Kreads [00:03, 37.1Kreads/s]198Kreads [00:03, 29.4Kreads/s]202Kreads [00:03, 22.9Kreads/s]258Kreads [00:03, 32.1Kreads/s]313Kreads [00:03, 44.8Kreads/s]374Kreads [00:03, 62.0Kreads/s]438Kreads [00:04, 85.1Kreads/s]496Kreads [00:04, 114Kreads/s] 555Kreads [00:04, 151Kreads/s]615Kreads [00:04, 194Kreads/s]674Kreads [00:04, 244Kreads/s]733Kreads [00:04, 296Kreads/s]797Kreads [00:04, 352Kreads/s]857Kreads [00:04, 402Kreads/s]916Kreads [00:04, 444Kreads/s]975Kreads [00:04, 464Kreads/s]
0.00reads [00:00, ?reads/s]694reads [00:00, 6.39Kreads/s]62.8Kreads [00:00, 9.09Kreads/s]129Kreads [00:00, 12.9Kreads/s] 196Kreads [00:00, 18.3Kreads/s]261Kreads [00:00, 25.8Kreads/s]328Kreads [00:00, 36.3Kreads/s]395Kreads [00:00, 50.6Kreads/s]461Kreads [00:00, 70.1Kreads/s]529Kreads [00:00, 95.8Kreads/s]599Kreads [00:01, 129Kreads/s] 664Kreads [00:01, 150Kreads/s]732Kreads [00:01, 196Kreads/s]796Kreads [00:01, 248Kreads/s]855Kreads [00:01, 301Kreads/s]915Kreads [00:01, 348Kreads/s]973Kreads [00:01, 393Kreads/s]
0.00reads [00:00, ?reads/s]2.07Kreads [00:00, 20.2Kreads/s]2.68Kreads [00:00, 9.97Kreads/s]4.98Kreads [00:00, 11.9Kreads/s]7.03Kreads [00:00, 13.5Kreads/s]11.1Kreads [00:00, 16.9Kreads/s]18.9Kreads [00:00, 22.0Kreads/s]22.7Kreads [00:02, 6.35Kreads/s]25.3Kreads [00:02, 5.36Kreads/s]27.3Kreads [00:03, 5.50Kreads/s]28.9Kreads [00:03, 5.67Kreads/s]30.2Kreads [00:03, 6.79Kreads/s]31.5Kreads [00:03, 7.71Kreads/s]32.7Kreads [00:03, 6.87Kreads/s]33.7Kreads [00:04, 6.11Kreads/s]34.6Kreads [00:04, 6.68Kreads/s]36.4Kreads [00:04, 8.21Kreads/s]37.6Kreads [00:04, 8.44Kreads/s]38.6Kreads [00:04, 7.14Kreads/s]39.5Kreads [00:04, 6.00Kreads/s]40.3Kreads [00:05, 4.64Kreads/s]40.9Kreads [00:05, 4.52Kreads/s]41.5Kreads [00:05, 4.68Kreads/s]42.0Kreads [00:05, 4.41Kreads/s]42.7Kreads [00:05, 4.86Kreads/s]43.3Kreads [00:05, 5.09Kreads/s]43.9Kreads [00:05, 5.33Kreads/s]44.6Kreads [00:05, 5.66Kreads/s]45.2Kreads [00:06, 5.45Kreads/s]45.9Kreads [00:06, 5.89Kreads/s]46.5Kreads [00:06, 5.63Kreads/s]47.3Kreads [00:06, 6.11Kreads/s]48.0Kreads [00:06, 5.91Kreads/s]48.6Kreads [00:06, 5.31Kreads/s]49.1Kreads [00:06, 4.50Kreads/s]49.6Kreads [00:06, 4.22Kreads/s]50.1Kreads [00:07, 4.24Kreads/s]50.7Kreads [00:07, 4.60Kreads/s]51.3Kreads [00:07, 5.00Kreads/s]51.8Kreads [00:07, 4.62Kreads/s]52.3Kreads [00:07, 4.53Kreads/s]52.8Kreads [00:07, 4.11Kreads/s]53.2Kreads [00:07, 3.99Kreads/s]53.6Kreads [00:07, 3.79Kreads/s]54.0Kreads [00:07, 3.72Kreads/s]54.4Kreads [00:08, 3.71Kreads/s]54.8Kreads [00:08, 3.70Kreads/s]55.3Kreads [00:08, 4.01Kreads/s]55.7Kreads [00:08, 3.99Kreads/s]56.1Kreads [00:08, 4.00Kreads/s]56.5Kreads [00:08, 3.87Kreads/s]57.0Kreads [00:08, 3.98Kreads/s]57.4Kreads [00:08, 4.00Kreads/s]57.8Kreads [00:08, 4.15Kreads/s]58.3Kreads [00:09, 4.27Kreads/s]58.8Kreads [00:09, 4.30Kreads/s]59.2Kreads [00:09, 3.96Kreads/s]59.6Kreads [00:09, 3.90Kreads/s]60.0Kreads [00:09, 3.95Kreads/s]60.5Kreads [00:09, 4.13Kreads/s]61.1Kreads [00:09, 4.60Kreads/s]61.6Kreads [00:09, 4.64Kreads/s]79.2Kreads [00:09, 6.55Kreads/s]100Kreads [00:09, 9.24Kreads/s] 125Kreads [00:10, 13.0Kreads/s]139Kreads [00:10, 17.5Kreads/s]161Kreads [00:10, 24.2Kreads/s]186Kreads [00:10, 33.2Kreads/s]204Kreads [00:10, 42.4Kreads/s]221Kreads [00:10, 45.8Kreads/s]234Kreads [00:10, 55.6Kreads/s]247Kreads [00:11, 62.5Kreads/s]258Kreads [00:11, 65.4Kreads/s]268Kreads [00:11, 56.8Kreads/s]281Kreads [00:11, 67.9Kreads/s]296Kreads [00:11, 81.2Kreads/s]311Kreads [00:11, 94.8Kreads/s]324Kreads [00:11, 89.2Kreads/s]353Kreads [00:12, 113Kreads/s] 397Kreads [00:12, 145Kreads/s]437Kreads [00:12, 179Kreads/s]477Kreads [00:14, 55.4Kreads/s]498Kreads [00:14, 69.1Kreads/s]519Kreads [00:14, 86.5Kreads/s]545Kreads [00:14, 108Kreads/s] 568Kreads [00:14, 129Kreads/s]590Kreads [00:14, 129Kreads/s]642Kreads [00:14, 166Kreads/s]697Kreads [00:14, 210Kreads/s]751Kreads [00:15, 257Kreads/s]814Kreads [00:15, 313Kreads/s]876Kreads [00:15, 367Kreads/s]939Kreads [00:15, 421Kreads/s]997Kreads [00:15, 457Kreads/s]
0.00reads [00:00, ?reads/s]30.6Kreads [00:00, 306Kreads/s]97.0Kreads [00:00, 365Kreads/s]163Kreads [00:00, 422Kreads/s] 230Kreads [00:00, 474Kreads/s]297Kreads [00:00, 521Kreads/s]364Kreads [00:00, 558Kreads/s]435Kreads [00:00, 595Kreads/s]504Kreads [00:00, 621Kreads/s]575Kreads [00:00, 646Kreads/s]643Kreads [00:01, 655Kreads/s]714Kreads [00:01, 672Kreads/s]784Kreads [00:01, 679Kreads/s]852Kreads [00:01, 668Kreads/s]920Kreads [00:01, 651Kreads/s]985Kreads [00:01, 630Kreads/s]
