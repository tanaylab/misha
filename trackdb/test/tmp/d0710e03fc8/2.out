2017-10-11 13:55:15 INFO::doing hamming 1
2017-10-11 13:55:18 INFO::removed 0 ambiguous barcodes
2017-10-11 13:55:18 INFO::doing hamming 1
2017-10-11 13:55:19 INFO::removed 0 ambiguous barcodes
2017-10-11 13:55:20 INFO::writing to /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/split/indexes/ZMillu7_S1_L002_001_indexes.tsv
2017-10-11 13:55:22 INFO::running /net/mraid14/export/tgdata/users/aviezerl/tools/R/3.4/gpatterns/import/map_fastq.py -r1 <(gzip -d -c /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/ZMillu7_S1_L002_R1_001.fastq.gz) -r2 <(gzip -d -c /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/ZMillu7_S1_L002_R2_001.fastq.gz) -i /net/mraid14/export/tgdata/users/aviezerl/proj/sc5mC/db/batches/BATCH24/PZM00020/fastq/ZMillu7/raw/split/indexes/ZMillu7_S1_L002_001_indexes.tsv
